### ISSUE TITLE
The compiled output includes `process.env`

### ISSUE THREAD

**Issue by ListenV** [aklinker1/vite-plugin-web-extension#96]
### Summary

The compiled output includes `process.env`. 

eg: `background.js` and `content_script.js`.

As far as I know, it's build in vite's Library Mode.

`process.env` variable not work in browser

### Environment

```
  System:
    OS: Windows 10 10.0.22621
    CPU: (16) x64 AMD Ryzen 7 4800H with Radeon Graphics
    Memory: 3.66 GB / 15.42 GB
  Binaries:
    Node: 18.15.0 - D:\Development\Scoop\apps\nodejs-lts\current\node.EXE
    npm: 9.5.0 - D:\Development\Scoop\apps\nodejs-lts\current\npm.CMD
  Browsers:
    Edge: Spartan (44.22621.1413.0), Chromium (111.0.1661.54)
    Internet Explorer: 11.0.22621.1
  npmPackages:
    vite: ^4.2.1 => 4.2.1
    vite-plugin-web-extension: ^3.0.2 => 3.0.2
```

**Comment by aklinker1** [comment]
I have seen this as well. It seems the build output changed since Vite 2, not surprising. I wasn't able to figure out why this happened, but library mode makes sense...

The workaround I'm currently using:

```ts
export default defineConfig({
  ...
  defines: {
    process.env: JSON.stringify({}),
  }
})
```

this works because, at least for me, `process.env` was the only value accessed on `process`.

I'll work on fixing the default config... Setting `build.target` seems like a likely fix.

**Comment by aklinker1** [comment]
Released in [`v3.0.4`](https://github.com/aklinker1/vite-plugin-web-extension/releases/tag/v3.0.4)
